src/test/java/net/sf/jabref/logic/xmp/XMPUtilTest.java

```diff
@@ -232,8 +236,7 @@ public void setUp() throws IOException, COSVisitorException {
     @After
     public void tearDown() {
         if (!pdfFile.delete()) {
-            System.err
-                    .println("Note: Cannot delete temporary file (already deleted so the corresponding test passed).");
+            LOGGER.info("Note: Cannot delete temporary file (already deleted so the corresponding test passed).");
```

Comment:
As you prefer. The TemporyFolder rule has the advantage, that the temp-file is directly deleted after the test (not sure what happens if I stop the test process mid-way if you use deleteOnExit).